sdk/schemaregistry/azure-schemaregistry-avroencoder/azure/schemaregistry/encoder/avroencoder/_abstract_avro_encoder.py

```diff
@@ -32,23 +29,22 @@ def get_schema_fullname(
     @abstractmethod
     def encode(
         self,
-        content,
-        schema,
-    ):
-        # type: (ObjectType, str) -> bytes
+        content: Mapping[str, Any],
+        schema: str,
+    ) -> bytes:
         """Convert the provided value to it's binary representation and write it to the stream.
         Schema must be a Avro RecordSchema:
         https://avro.apache.org/docs/1.10.0/gettingstartedpython.html#Defining+a+schema
         :param content: An object to encode
-        :type content: ObjectType
+        :type content: Mapping[str, Any]
         :param schema: An Avro RecordSchema
         :type schema: str
         :returns: Encoded bytes
         :rtype: bytes
         """
 
     @abstractmethod
-    def decode(self, content: Union[bytes, BinaryIO], reader: Any):
+    def decode(self, content: Union[bytes, BinaryIO], reader: Any) -> Dict[str, Any]:
         """Read the binary representation into a specific type.
         Return type will be ignored, since the schema is deduced from the provided bytes.
         :param content: A stream of bytes or bytes directly
```

Comment:
the docstring is not matching the function signature right, missing **kwargs as well  ?

Reply:
updated!